alicloud/table_alicloud_oss_bucket.go

```diff
@@ -75,6 +75,13 @@ func tableAlicloudOssBucket(ctx context.Context) *plugin.Table {
 				Transform:   transform.FromField("BucketInfo.SseRule").Transform(bucketSSEConfiguration),
 				Description: "The server-side encryption configuration for bucket",
 			},
+			{
+				Name:        "lifecycle_configuration",
```

Comment:
@Subhajit97 The column name `lifecycle_configuration` makes sense, but I'm wondering how do determine if we should add the word `configuration` on or not? Based on the API call, it looks like they just call it `lifecycle`, while TF has `lifecycle_rule` resources.

Are these a collection of lifecycle rules, if so, should we go with the column name `lifecycle_rules`? The PR description doesn't have any examples, so I'm not sure what these look like. Can you please include them?)